README.md

```diff
@@ -94,9 +94,9 @@ Instructions for developers for testing [here](docs/TESTING.md). If you want to
 
 ### Community
 
-+ IRC: `#joinmarket` on irc.freenode.net https://webchat.freenode.net/?channels=%23joinmarket (logs can be found [here](http://gnusha.org/joinmarket/))
++ IRC: `#joinmarket` on irc.libera.chat https://kiwiirc.com/nextclient/irc.libera.chat#joinmarket (logs can be found [here](http://gnusha.org/joinmarket/))
 
-+ IRC on tor: `#joinmarket` on the networks [Hackint](https://www.hackint.org/) and [Darkscience](https://www.darkscience.net/). These channels are bridged to the above freenode channel.
++ IRC on tor: `#joinmarket` on the network [Hackint](https://www.hackint.org/). This channel is bridged to the above libera channel.
```

Comment:
```suggestion
IRC over Tor: `#joinmarket` on the [Hackint](https://www.hackint.org/) network, which is bridged to the above libera.chat channel.
```

Pure bikeshedding lol. Is "libera" the official shorthand for libera.chat? Personally I prefer the full libera.chat, but maybe that part is OK as-is.

Reply:
Yes. Checked their website again and their official name is "Libera Chat". Lets go with this.